event-based-simulators/main/event_based_simulators.py

```diff
@@ -107,17 +107,16 @@ def add_Shiftplan_to_OEE(self):
 
 
     def __create_task(self):   
-        task = PeriodicTask(shiftplan_polling_interval, shiftplan_polling_interval, self.getShiftplan)
-        
-        log.debug(f'create periodic task for pulling shiftplans running every {shiftplan_polling_interval}')      
+        task = PeriodicTask(shiftplan_polling_interval, shiftplan_polling_interval, self.getShiftplan())
```

Comment:
Btw this line is wrong. We don't want to pass the return value of getShiftplan, but we want to pass the function itself. So we falsely merged this. I will revert this change on https://github.com/SoftwareAG/oee-simulators/pull/33